group_vars/gxconfig.yml

```diff
@@ -286,6 +283,15 @@ base_app_main: &BASE_APP_MAIN
   # <config_dir>.
   tool_sheds_config_file: "{{ galaxy_config_dir }}/tool_sheds_conf.xml"
 
+  # This option controls whether legacy datatypes are loaded from
+  # installed tool shed repositories. We're are in the process of
+  # disabling Tool Shed datatypes. This option with a default of true
+  # will be added in 22.01, we will disable the datatypes on the big
+  # public servers during that release. This option will be switched to
+  # False by default in 22.05 and this broken functionality will be
+  # removed all together during some future release.
+  #load_tool_shed_datatypes: true
```

Comment:
I guess we can switch this to false later this week and see how many people cry - I expect none.